javascript/organizing_your_javascript_code/classes.md

```diff
@@ -36,7 +36,7 @@ This section contains a general overview of topics that you will learn in this l
 
 4. Classes can have [static properties and methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static) which are properties and methods that are accessed on the class itself and not on the instance of a class. This is similar to how some string methods are accessed on the instance of a string itself e.g. `someString.slice(0, 5)` whereas some methods are called on the String constructor directly e.g. `String.fromCharCode(79, 100, 105, 110)`.
 
-5.  [This article](https://medium.com/@rajaraodv/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65) provides some pros and cons for classes. [FunFunFunction's video on Composition over Inheritance](https://www.youtube.com/watch?v=wfMtDGfHWpA) elaborates on the cons mentioned in the article and does a great job of going over the topic. Many people think that class syntax is misleading for JavaScript and Inheritance is best avoided, and thus Factory Functions (from the previous lesson) that use Composition are inherently better. __We are NOT saying that classes are bad or that inheritance should never be used!__ We just want you to be informed of the opinions of both sides.
+5.  [This article](https://medium.com/@rajaraodv/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65) provides some pros and cons for classes. [FunFunFunction's video on Composition over Inheritance](https://www.youtube.com/watch?v=wfMtDGfHWpA) elaborates on the cons mentioned in the article and does a great job of going over the topic.
```

Comment:
```suggestion
5.  Read [this article covering opinions regarding the pros and cons of classes](https://medium.com/@rajaraodv/is-class-in-es6-the-new-bad-part-6c4e6fe1ee65). [FunFunFunction's video on Composition over Inheritance](https://www.youtube.com/watch?v=wfMtDGfHWpA) elaborates on the cons mentioned in the article and does a great job of going over the topic.
```

I would like to propose this change to make the link more descriptive. I do think that including the word "opinions" could be helpful here since that's what we're dealing with. Let me know what you think!

Reply:
Sounds good! I can start working on it tomorrow. Shall I wait for more input before starting?

Reply:
@Ariech You don't have to wait :) You can also click the `Commit Suggestion` button if you want to directly commit my proposal. That will add a new commit right here without you having to locally edit the file.

Reply:
@ManonLef Done, thank you for the help! The `Commit Suggestion` seems like a neat feature :)